dynamic-plugins.default.yaml

```diff
@@ -575,6 +575,22 @@ plugins:
                     anyOf:
                     - isNexusRepositoryManagerAvailable
 
+  - package: ./dynamic-plugins/dist/janus-idp-backstage-plugin-acr
+    disabled: true
+    pluginConfig:
+      dynamicPlugins:
+        frontend:
+          janus-idp.backstage-plugin-acr:
+            mountPoints:
+              - mountPoint: entity.page.image-registry/cards
+                importName: AcrPage
+                config:
+                  layout:
+                    gridColumn: 1 / -1
+                  if:
+                    anyOf:
+                    - isAcrAvailable
+
```

Comment:
Seems to me that the [proxy config](https://github.com/janus-idp/backstage-showcase/pull/734/files#diff-ec52f22d476ccc33271d11c4f08a68369614378aa0cb9aa5aba2f08943cd68dfR102) should also be added here.
But it's something that needs to be done for other plugins as well, so it could be done as a followup PR later on.